server/db/driver/pg/upgrades.go

```diff
@@ -291,6 +295,30 @@ func v4Upgrade(tx *sql.Tx) (err error) {
 	return err
 }
 
+func v5Upgrade(tx *sql.Tx) (err error) {
+	mkts, err := loadMarkets(tx, marketsTableName)
+	if err != nil {
+		return fmt.Errorf("failed to read markets table: %w", err)
+	}
+
+	doTable := func(tableName string) error {
+		_, err = tx.Exec(fmt.Sprintf("ALTER TABLE %s ADD COLUMN epoch_gap INT4 DEFAULT -1;", tableName))
+		return err
+	}
+
+	log.Info("Adding epoch_gap column to %d market tables", len(mkts)*2)
```

Comment:
Info**f**

Also, this reads a bit like you have N markets.  But since the active cancels table will be instant compared to the _archived_ table, I'd prefer if this read `log.Infof("Adding epoch_gap column to cancel tables for %d markets", len(mkts))`